src/main/webapp/pages/projects/project_new.html

```diff
@@ -213,6 +220,32 @@ <h2 class="panel-title" th:text="#{projects.create.heading}">Create a New Projec
           }
         });
 
+        /*
+            Disables/enables the lockSamples checkbox
+            depending on if using the samples from cart
+        */
+        $("#useCartSamples").on("change", function(e) {
+          var selected = $(e.target).prop("checked");
+          if (!selected) {
+            $("#lockSamples").attr("disabled", true);
+          } else {
+            $("#lockSamples").removeAttr("disabled");
```

Comment:
Add a `$("#lockSamples").prop("checked", false)` above when you disable the checkbox to automatically uncheck it too please.

Reply:
Fixed in 94d58a64cfc89091be01471a00966a4ad200ad8e